lgc/interface/lgc/CommonDefs.h

```diff
@@ -30,8 +30,13 @@
  */
 #pragma once
 
+#include <array>
+
 namespace lgc {
 
+// Type used to hold a 128-bit hash value in LGC and LLPC.
+using Hash128 = std::array<unsigned char, 16>;
```

Comment:
I've seen `uint8_t` used to represent blobs in other parts of the ICD. Can we stick to that type here?

Reply:
I was trying to avoid including `cstdint` just to use `uint8_t`.  However, you are right that `uint8_t` is the better choice, so I made the change.